zerver/views/pointer.py

```diff
@@ -52,3 +52,12 @@ def get_profile_backend(request, user_profile):
         result['max_message_id'] = messages[0].id
 
     return json_success(result)
+
+def deactivate_user_backend(request, user_profile):
+    # type: (HttpRequest, UserProfile) -> HttpResponse
+    return _deactivate_user_profile_backend(request, user_profile)
+
+def _deactivate_user_profile_backend(request, target):
+    # type: (HttpRequest, UserProfile) -> HttpResponse
+    do_deactivate_user(target)
+    return json_success({})
```

Comment:
At the very least move this to be in `users.py` so that it's obvious to someone that they can later refactor this to share code with the existing code for an admin deactivating another user... 

`get_admin_users` in `zerver/models.py` can get you the list of administrators in an organization (`git grep` is a great tool for finding things like this).

BTW I'd add a test for this once you've done it; I think "iago@zulip.com" is the only realm admin in the default test data, so you should be able to write a test where Iago tries to remove himself, check that fails, and then Iago makes Hamlet an admin and can deactivate himself.